R-packages/covidcast/R/plot.R

```diff
@@ -346,22 +350,27 @@ plot_bubble = function(x, time_value = NULL, include = c(), range = NULL,
   geom_args$data = map_df
   polygon_layer = do.call(ggplot2::geom_polygon, geom_args)
 
-  # Set the lats and lons for counties
+  # Retrieve coordinates for mapping
+  # Reading from usmap files to ensure consistency with borders
   if (attributes(x)$geo_type == "county") {
-    g = county_geo[county_geo$FIPS %in% map_geo, ]
-    cur_geo = g$FIPS
-    cur_lon = g$LON
-    cur_lat = g$LAT
+    centroid_file = system.file("extdata",
```

Comment:
@JedGrabman Instead of reading something from csv each time this gets called, you could do this one, back out in the data-raw/make.R file, and overwrite the `county_geo` and `state_geo` data frames that get defined there, and that get loaded when the `covidcast` package is loaded.  

I imagine this is the preferred route, but do you see any reasons not to do it the way I'm proposing?  Also tagging @capnrefsmmat.

Reply:
Yeah, I think the best options would be (a) get these into a dataset provided with the package, like county_geo or state_geo, or (b) if usmap exports this as a data frame instead of just a CSV, import that from usmap.

Reply:
@ryantibs 
Agreed, that seems like the correct approach.

@capnrefsmmat 
a.) Sounds good  
b.) Unfortunately, I couldn't find anywhere that usmap makes these available as a dataframe. That would be preferable, because then we wouldn't be relying on the internal implementation of usmap. Without that, it seems our choice is manually curating the location data vs. relying on usmap's csv. Creating the geo files in make.R should mitigate, but not entirely eliminate, the risk from reading from the csv.

Reply:
After you update make.R, you only need to run it once to population the data files (currently `county_geo`, `state_geo`, etc.) that go into the package.  (That is, I'm just clarifying to @JedGrabman that it's not dynamically run each time on package load.)  So the only risk is that if usmap updates its shape files for drawing the map, and we don't catch it and don't rerun make.R, then the given `county_geo` and `state_geo` data sets in covidcast will be stale.  But once we catch it, we can just rerun make.R.  Since this shouldn't happen very often, this seems like a low-risk approach to me.

Reply:
Sounds good. I've replaced the `county_geo` and `state_geo` files. This change is not backwards compatible if somebody is directly using those files.

Reply:
Looks good to me.  Thanks!  I will merge the PR now.